packaging/buildTasks.mjs

```diff
@@ -124,6 +124,7 @@ export async function packResources() {
     const start = Date.now()
 
     const contracts = await glob("contractdata/**/*.json")
+    contracts.sort()
```

Comment:
Thanks for the PR! I'm not sure if this actually addresses the root issue though, as `.sort` will just be calling `toString` on each object, which should theoretically be `[object Object]`. Therefore, I think we should explicitly define a sorting factor here, something like `contractData?.Metadata?.Id`

Reply:
`contracts` is a string array, this just sorts the files alphabetically